xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/DeleteAction.java

```diff
@@ -136,6 +138,14 @@ protected boolean delete(XWikiContext context) throws XWikiException
 
         String sindex = request.getParameter(RECYCLED_DOCUMENT_ID_PARAM);
         String emptyBin = request.getParameter(EMPTY_RECYCLE_BIN);
+        // the document skips the recyclebin only if the user has explicitly made the choice to skip it
```

Comment:
Uppercase to start sentences in English ;)

Reply:
Right, fixing on the whole PR.